### PR TITLE
Update modal.js

### PR DESCRIPTION
**Pull request by Maddesto**
close modal only if it is open

### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title
2. ...

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=17665) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/magento2?pullRequest=17665) before we can accept your contribution.<br/>

**Comment by magento-engcom-team** [comment]
Hi @Maddesto. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me {$VERSION} instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by VladimirZaets** [comment]
Hi @Maddesto, please, sign CLA, otherwise, we can't process your pull request.

File: app/code/Magento/Ui/view/base/web/js/modal/modal.js [hunk 1]
```diff
@@ -305,6 +305,9 @@ define([
          * * @return {Element} - current element.
          */
         closeModal: function () {
+            if (!this.options.isOpen) {
+                return this.element;
+            }
             var that = this;
```
Comment (VladimirZaets) [inline app/code/Magento/Ui/view/base/web/js/modal/modal.js]: According to Magento code style guideline, the variables declaration should be in the top of the function. Please move `var` declaration to the top of the function.

**Comment by sidolov** [comment]
Hi @Maddesto , I'm closing this PR due to missing signed CLA. 
Please, reopen it and sign CLA if you wish to continue. 
Thank you for the contribution!

**Pull request closed without merging**